### PR TITLE
Add application review and submit screens

### PR DESCRIPTION
**Pull request by maxious**
@nathf how do I make it redirect to a different page at the end?

### CODE REVIEW

File: src/bundles/SellerRegistration/components/Review.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                <figure></figure>
+                <article>
+                    <p>
+                        <strong>ABN</strong><br/>
+                        { data.abn }
+                    </p>
+
+                    <p>
+                        <strong>Website</strong><br/>
+                        <a href="{ data.website }" target="_blank">{ data.website }</a>
```
Comment (matt-sm) [inline src/bundles/SellerRegistration/components/Review.js]: that href does not work

File: src/bundles/SellerRegistration/components/Review.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                    <p>
+                        { data.address.addressLine }<br/>
+                        { data.address.suburb }<br/>
+                        { data.address.state } { data.address.postalCode }
+                    </p>
+                </article>
+            </li>
+        </ul>
+        <p>
+            <a role="button" href={submit} onClick={onClick}>Confirm</a>
```
Comment (matt-sm) [inline src/bundles/SellerRegistration/components/Review.js]: are you missing type="submit" ?

Comment (matt-sm) [inline src/bundles/SellerRegistration/components/Review.js]: Button should be Save & Continue like the previous buttons

File: src/bundles/SellerRegistration/components/Submit.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        <p>
+            <a role="button" href={signup} onClick={onClick}>Submit Application </a>
+        </p>
+    </div>
+);
+
+Submit.defaultProps = {
+    onClick: () => {
+    },
+    signup: '#'
```
Comment (matt-sm) [inline src/bundles/SellerRegistration/components/Submit.js]: signupUrl might make more sense?

File: src/bundles/SellerRegistration/components/Review.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+};
+
+const mapStateToProps = (state) => {
+    return {
+        data: flattenStateForms(state)
+    }
+}
+
+const matchDispatchToProps = (dispatch) => {
+    return bindActionCreators({Review}, dispatch)
```
Comment (nathf) [inline src/bundles/SellerRegistration/components/Review.js]: How is this working?

File: src/bundles/SellerRegistration/components/Review.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+                        { data.abn }
+                    </p>
+
+                    <p>
+                        <strong>Website</strong><br/>
+                        <a href="{ data.website }" target="_blank">{ data.website }</a>
+                    </p>
+                    <p>
+                        <strong>LinkedIn</strong><br/>
+                        <a href="{ data.linkedin }" target="_blank">{ data.linkedin }</a>
```
Comment (nathf) [inline src/bundles/SellerRegistration/components/Review.js]: These attributes shouldnt be wrapped in quotes

File: src/bundles/SellerRegistration/components/Review.js (hunk 1 above)
Comment (nathf) [inline src/bundles/SellerRegistration/components/Review.js]: These attributes shouldnt be wrapped in quotes

File: src/bundles/SellerRegistration/ReviewWidget.js [hunk 6]
```diff
@@ -0,0 +1,8 @@
+import React from 'react'
```
Comment (nathf) [inline src/bundles/SellerRegistration/ReviewWidget.js]: Do these need to have widgets? Will they be used outside the signup flows?

File: src/bundles/SellerRegistration/SubmitWidget.js [hunk 7]
```diff
@@ -0,0 +1,8 @@
+import React from 'react'
```
Comment (nathf) [inline src/bundles/SellerRegistration/SubmitWidget.js]: Do these need to have widgets? Will they be used outside the signup flows?

**Comment by nathf** [comment]
@maxious Just use a plain anchor to redirect somewhere else

File: src/bundles/SellerRegistration/components/Review.js [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+                        { data.abn }
+                    </p>
+
+                    <p>
+                        <strong>Website</strong><br/>
+                        <a href={data.website} target="_blank">{ data.website }</a>
+                    </p>
+                    <p>
+                        <strong>LinkedIn</strong><br/>
+                        <a href={data.linkedin} target="_blank">{ data.linkedin }</a>
```
Comment (nathf) [inline src/bundles/SellerRegistration/components/Review.js]: these external links will also need `rel="external"`

**Pull request merged**